### ISSUE TITLE
Review MISSING annotations to GO:0000165 MAPK cascade

### ISSUE THREAD

**Issue by sabrinatoro** [geneontology/go-annotation#1795]
When reviewing annotations for the MAPK signaling project, we found that some genes were annotated to Molecular Function terms which are part of the MAPK cascade (for example: MAP Kinase activity-GO:0004707, MAP kinase kinase activity-GO:0004708, MAP kinase kinase kinase activity-GO:0004709), but did not have annotation to the Biological Process “MAPK cascade” (GO:0000165).

Annotations for these genes should be reviewed, and annotation to the BP “MAPK cascade” (GO:0000165), or a more specific kinds of MAPK pathway (e.g. ERK5 cascade (GO:0070375), ERK1 and ERK2 cascade (GO:0070371)) , should be added when appropriate. 

Genes/annotations to review are here: 
https://docs.google.com/spreadsheets/d/1tGxb6tNGH5oecXwLNZ4SDHEzNPDwRUfiE-m5v2e05Go/edit?usp=sharing 

Notes: 
- This list of genes/annotations is based on annotations downloaded on Jul 13, 2017. The annotations were reviewed and compared in order to give a list of genes which did have MF annotation to MAPK signaling related activity, but did not have BP annotation to the cascade. 
- Annotations for individual genes were highlighted with the same color (apologies for the highly colorful document, feel free to remove the colors if you want).

**Comment by sabrinatoro** [comment]
Number of genes to review per group:
Groups: #genes/annotation to add / color coded by genes
AgBase: 1
AspGD:1
BHF-UCL: 1
CACAO: 1
CGD: 10
FlyBase: 2
GeneDB: 6
MGI: 1
PAMGO_MGG: 5
Reactome: 1
RGD: 8
SGD: 12
TAIR: 4
TIGR:1
Uniprot: 19
WB: 4
ZFIN: 4

**Comment by deustp01** [comment]
Agreed that in the one Reactome case absence of a BP term counts as missing information, but the way that we create BP annotations does not provide any easy way to add it here. (NOT an argument against the basic idea that the MF implies the BP.)
Sorry

**Comment by srengel** [comment]
Hi @sabrinatoro, do the colors in the spreadsheet mean anything?

**Comment by sabrinatoro** [comment]
@srengel : 
each gene has the same color (some gene has multiple annotations). 
The colors themselves do not have any meaning.

**Comment by tberardini** [comment]
> we found that some genes were annotated to Molecular Function terms which are part of the MAPK cascade 

Would it make sense to link the MF terms (for example: MAP Kinase activity-GO:0004707, MAP kinase kinase activity-GO:0004708, MAP kinase kinase kinase activity-GO:0004709) in question by **_part_of_** to the BP “MAPK cascade” (GO:0000165). Then the BP annotation could be inferred by the existing MF-BP inference pipeline.  

This means addressing the situation by ontology editing instead of annotation editing.

**Comment by ValWood** [comment]
> This means addressing the situation by ontology editing instead of annotation editing.

was just about to suggest the same...

**Comment by ukemi** [comment]
Those MF->BP links already exist. If people are happy with that, then we are ok.

**Comment by srengel** [comment]
negative regulation of MAPK cascade
activation of MAPK activity

**Comment by ValWood** [comment]
Are the F-P links working though, we have opened some tickets a while ago because we suspected that they weren't. The ticket isn't addressed yet:
https://github.com/geneontology/go-annotation/issues/1683

(presumably if they were working you would see the annotations with the same evidence code)

**Comment by RLovering** [comment]
Hi 
sorry to be a pain but the definition and comments associated with GO:0070371 ERK1 and ERK2 cascade are confusing:
Definition:An intracellular protein kinase cascade containing at least ERK1 or ERK2 (MAPKs), a MEK (a MAPKK) and a MAP3K. The cascade can also contain two additional tiers: the upstream MAP4K and the downstream MAP Kinase-activated kinase (MAPKAPK). The kinases in each tier phosphorylate and activate the kinases in the downstream tier to transmit a signal within a cell. PMID:20811974
comments:Note that this MAPKKK cascade is commonly referred to as the ERK pathway in the literature, but involves only ERK1 or ERK2, not other ERK kinases.

I think this means that MAP4K and MAPKAPK should both be annotated to GO:0070371 ERK1 and ERK2 cascade, rather than regulating the GO:0070371 ERK1 and ERK2 cascade.

If this is correct maybe these could be revised to:
Definition:An intracellular protein kinase cascade containing at least ERK1 or ERK2 (MAPKs), a MEK (a MAPKK) and a MAP3K. **The cascade may involve 4 different kinases, as it** can also contain two additional tiers: the upstream MAP4K and the downstream MAP Kinase-activated kinase (MAPKAPK). The kinases in each tier phosphorylate and activate the kinases in the downstream tier to transmit a signal within a cell. PMID:20811974

comments:Note that **the** MAPKKK cascade is commonly referred to as the ERK pathway in the literature, but **the ERK pathway is often only part of the ERK1 and ERK2 cascade** as it involves only ERK1 or ERK2, and not other ERK kinases.

I have tried to indicate the suggested revisions using bold highlights

PS I have updated the annotation listed for BHF from positive regulation of ERK1 and ERK2 cascade to ERK1 and ERK2 cascade

Ruth
